gencred/merge_kubeconfig_secret.py

```diff
@@ -44,11 +50,31 @@ def call(cmd, **kwargs):
         shell=True,
         stderr=sys.stderr,
         stdout=subprocess.PIPE,
-        timeout=10,#seconds
+        timeout=10,  # seconds
         universal_newlines=True,
         **kwargs,
     )
 
+
+def save_secret_to_gcp(project_id, data, secret_id=gcp_secret_key):
+    dst = f"projects/{project_id}/secrets/{secret_id}"
+    print('Saving secret to %s' % dst)
+    client = secretmanager.SecretManagerServiceClient()
```

Comment:
Huh, no auth needed?

Reply:
Actually it's needed, it requires ADC auth which is done by gcloud, I have added the required command in the description below, as well as `requirement` section on top